recipe/migrations/example.exyaml

```diff
@@ -0,0 +1,4 @@
+__migrator: kind: version
```

Comment:
Should this file be in `recipe`? This doesn't belong here IMO.

I think this should go in `.ci_support`.

Reply:
@mariusvniekerk raised the issue that rerenders might do bad things to this. He might have more info on this though.

Reply:
That can be fixed easily by not deleting this folder.

Reply:
Can you put in a PR/edits into the conda-smithy PR to change where it looks for the migrations?